### ISSUE TITLE
Add timeline trace chart

### ISSUE THREAD

**Issue by markrey** [google/timesketch#728]
Feature request :

Currently there are two charts for showing events:
 
<img width="1280" alt="timeline" src="https://user-images.githubusercontent.com/8405940/49346581-6e65a200-f649-11e8-9577-023c7158c423.png">

One of the very good chart would be Trace Chart - For example OpenTracing type chart 

![image](https://user-images.githubusercontent.com/8405940/49346599-b684c480-f649-11e8-96f6-d16a2a51bb68.png)

For example:
This chart allows users to quickly check the journey of the attack (in our usecase). We take one email address as reference and see when they logged in successfully from which PC to how many times a user failed login across multiple PCs.

Thank you

**Comment by berggren** [comment]
Yes, this is a great idea! I have been thinking about something similar. Essentially we want some sort of Gant chart.

**Comment by berggren** [comment]
Assigning to @sydp who is working on the viz in the new UI

**Comment by sydp** [comment]
The trace chart can probably be implemented using the Apex rangeBar and an aggregator that gets the min-max timestamp for top-n terms.
